docs/ibc/proposals.md

```diff
@@ -92,30 +90,20 @@ If the chain has been updated to ibc-go >= v1.1.0, anyone can submit the governa
   {
     "messages": [
       {
-        "@type": "/ibc.core.client.v1.ClientUpdateProposal",
-        "title": "title_string",
-        "description": "description_string",
+        "@type": "/ibc.core.client.v1.MsgRecoverClient",
         "subject_client_id": "expired_client_id_string",
-        "substitute_client_id": "active_client_id_string"
+        "substitute_client_id": "active_client_id_string",
```

Comment:
I haven't verified manually. If someone thinks that is important, I'll let you take the lead :slightly_smiling_face: 

I did use the gov cli cmd to format this:
```
Example:
$ simd tx gov submit-proposal path/to/proposal.json

Where proposal.json contains:

{
  // array of proto-JSON-encoded sdk.Msgs
  "messages": [
    {
      "@type": "/cosmos.bank.v1beta1.MsgSend",
      "from_address": "cosmos1...",
      "to_address": "cosmos1...",
      "amount":[{"denom": "stake","amount": "10"}]
    }
  ],
  // metadata can be any of base64 encoded, raw text, stringified json, IPFS link to json
  // see below for example metadata
  "metadata": "4pIMOgIGx1vZGU=",
  "deposit": "10stake",
  "title": "My proposal",
  "summary": "A short summary of my proposal",
  "expedited": false
}
```

Reply:
wouldn't this just be `simd tx ibc client "subject-client" "substitute-client" --authority="..." --title="..."`

this suggestion comes after looking at sdk's [docs for x/upgrade ](https://docs.cosmos.network/main/modules/upgrade#transactions) which uses gov v1.

Reply:
There's 2 cli's that support submission of recovering a client. Using gov's submit-proposal cli and the one we implemented. I actually ended up testing recovery via the gov cli using the e2e targetting cli's. Personally I prefer the json file approach, otherwise you input all the args as flags